### PR TITLE
Fixed roary to include inverted quotes around input filenames

### PR DESCRIPTION
**Pull request by Slugger70**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

See issue #2545 

Hopefully this provides a fix.

### CODE REVIEW

File: tools/roary/roary.xml [hunk 1]
```diff
@@ -23,10 +28,12 @@
           -t '$advanced.trans_tab'
           -iv '$advanced.mcl'
 
-          *.gff
-          
+          #for $f in $filenames
+            "$f"
```
Comment (bgruening) [inline tools/roary/roary.xml]: single quotes

File: tools/roary/roary.xml [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
     <requirements>
       <requirement type="package" version="3.12.0">roary</requirement>
     </requirements>
-    
+
     <command detect_errors="exit_code"><![CDATA[
-        
         #for $gff in $gff_input.gffs
-          cp $gff '${gff.element_identifier}'.gff &&
+          cp $gff "${gff.element_identifier}.gff" &&
```
Comment (bgruening) [inline tools/roary/roary.xml]: please use single-quotes wherever you can.

Also wouldn't it be better to replace spaces, like here: https://github.com/galaxyproject/tools-iuc/blob/57493fdc88e6128bed127993464f2a6480e2774d/tools/mothur/filter.seqs.xml#L40

**Comment by bgruening** [comment]
ping @Slugger70

File: tools/roary/roary.xml (hunk 2 above)
Comment (Slugger70) [reply tools/roary/roary.xml]: I think allowing for spaces in general is more fool proof.. :)

File: tools/roary/roary.xml [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
     <requirements>
       <requirement type="package" version="3.12.0">roary</requirement>
     </requirements>
-    
+
     <command detect_errors="exit_code"><![CDATA[
-        
         #for $gff in $gff_input.gffs
-          cp $gff '${gff.element_identifier}'.gff &&
+          cp $gff '${gff.element_identifier}.gff' &&
```
Comment (bgruening) [inline tools/roary/roary.xml]: gff should be quoted imho

File: tools/roary/roary.xml [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
+        #end for
+
+        #set $filenames = list()
+        #for $gff in $gff_input.gffs
```
Comment (bgruening) [inline tools/roary/roary.xml]: can both loops not be merged?

**Comment by abretaud** [comment]
Do you have time to make the little changes @Slugger70?

**Comment by bgruening** [comment]
He is on an Admin workshop the entire week, I doubt so ...

**Pull request merged**